# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/seatd/video-group.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- seatd-0.9.2/contrib/systemd/seatd.service.vanilla	2026-01-10 13:59:42.328593873 +0100
+++ seatd-0.9.2/contrib/systemd/seatd.service	2026-01-10 13:59:47.420673809 +0100
@@ -5,7 +5,7 @@
 [Service]
 Type=simple
 # Specify the group you'd like to grant access to seatd
-ExecStart=seatd -g seat
+ExecStart=seatd -g video
 Restart=always
 RestartSec=1
 
